rewards/aws/trees.py

```diff
@@ -13,7 +14,7 @@ def download_latest_tree(chain: str):
     Download the latest merkle tree that was uploaded for a chain
     :param chain: the chain from which to fetch the latest tree from
     """
-    if chain == "eth":
+    if chain == Network.Ethereum:
```

Comment:
`badger-tree.json` should match formatting for other chains, such as `badger-tree-ethereum.json`.

Will add this as a ticket as it's out of scope.